src/components/Dashboard.tsx

```diff
@@ -222,7 +222,9 @@ export const Dashboard = ({
         <>
           <DataCard>
             <Title mt={10} order={2}>
-              {t("dashboard.timePerDay")}
+              {dayCount > 180
```

Comment:
Please create a new constant for this, I see it's used at least here and a few lines below, possibly in more places also.